odoo/tools/translate.py

```diff
@@ -250,11 +250,15 @@ def process(node):
                 translated = callback(original)
                 if translated:
                     result = content.replace(original, translated)
-                    div = parse_html(f"<div>{result}</div>")
-                    if pos:
-                        node[pos-1].tail = div.text
-                    else:
-                        node.text = div.text
+                    # <div/> is used to auto fix crapy result
+                    # <span/> is one of TRANSLATED_ELEMENTS and used to check translatable
+                    result_span = parse_html(f"<div><span>{result}</span><div>")[0]
```

Comment:
This variable name is confusing: it is an HTML `div`, but you call it a `span` :confused: 
Also, I don't understand how the `span` is added here, but seem to disappear from the result...

Reply:
`parse_html(f"<div><span>{result}</span><div>")` is a div
`parse_html(f"<div><span>{result}</span><div>")[0]`  is equal to `parse_html(f"<span>{result}</span>")`  if the `result` is not crapy

In the latest code, I change the tag from `'div'`  to `'span'` as a hack
The reason why I don't use `parse_html(f"<span>{result}</span>")` directly
if the `result = 'a</span>b</span>c'`(we do have this kind of bad translation in the po file) the `parse_html(f"<span>{result}</span>")` will have error